tpm/tpm.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package tpm supports direct communication with a tpm device under Linux.
+// tpmutil.Package tpm supports direct communication with a tpm device under Linux.
```

Comment:
I think this is a copy-paste error (s/pack/tpmutil.Pack/gi ?). This comment should start with "Package tpm"

Reply:
Done